README.rst

```diff
@@ -117,14 +117,14 @@ Using tox
 
 .. code-block:: bash
 
-    $ tox
+    $ python -m tox
```

Comment:
Actually we are not using tox ourselves at the moment, maybe remove it @jgsogo ?

Reply:
no idea, but I've seen it was used in examples.

Reply:
For Conan itself we are not using it. If it works we can leave it (some users submitting PRs have said they have successfully tested it using tox, but I've never tried it )

Reply:
tox is an amazing tool to configure the Python environment in which you are going to test your application: it helps you to use the proper python version, the checkout of the app you want to test, and it also cleans the environment variables so they don't interfere in your tests... that's why we are using it in the examples and the hooks one.